metacat-metadata-mysql/src/main/java/com/netflix/metacat/metadata/mysql/MysqlUserMetadataService.java

```diff
@@ -856,10 +860,26 @@ public void populateOwnerIfMissing(final HasDefinitionMetadata holder, final Str
         SearchMetadataQuery buildSearchMetadataQuery(@Nullable final Set<String> propertyNames,
                                                      @Nullable final String type,
                                                      @Nullable final String name,
-                                                     @Nullable final String sortBy,
-                                                     @Nullable final String sortOrder,
+                                                     @Nullable final String sortByStr,
+                                                     @Nullable final String sortOrderStr,
                                                      @Nullable final Integer offset,
                                                      @Nullable final Integer limit) {
+            String sortBy = null;
+            if (StringUtils.isNotBlank(sortByStr)) {
+                sortBy = sortByStr.trim().toLowerCase();
+                if (!DEFINITION_METADATA_SORT_BY_COLUMNS.contains(sortBy)) {
```

Comment:
What if this is a comma separated list of columns?

Reply:
Let's not support comma separated list for now.

Reply:
ok